lib/senkyoshi/models/file_resource.rb

```diff
@@ -19,5 +19,19 @@ def self.from(xml, pre_data, _resource_xids = nil)
     def iterate_xml(_xml, _pre_data)
       self
     end
+
+    def create_module(course)
+      course.canvas_modules ||= []
+      cc_module = Course.master_module(course)
+      if cc_module
+        cc_module.module_items << @module_item
+      else
+        cc_module = Module.new(MASTER_MODULE, MASTER_MODULE)
```

Comment:
What would you think about making a new module default values to `MASTER_MODULE`? So you can just say `cc_module = Module.new`?